docs/multicluster/user-guide.md

```diff
@@ -386,7 +387,7 @@ there are several possibilities:
 * You can choose to use the K8s Node's `ExternalIP` as `gatewayIP`, by changing
 the configuration option `gatewayIPPrecedence` to value: `external`, when
 deploying the member Multi-cluster Controller. The configration option is
-defined in ConfigMap `antrea-mc-controller-config-***` in `antrea-multicluster-member.yml`.
+defined in ConfigMap `antrea-mc-controller-config` in `antrea-multicluster-member.yml`.
```

Comment:
Hi @jianjuns  Could you take a look again? I have re-based the code to include pod-to-pod doc change.
This config name is updated by #4187 (Add checksum of config file as an annotation), so all suffix is removed in this PR.